api/tests/test_submit_job.py

```diff
@@ -99,3 +99,15 @@ def test_submit_job_granule_does_not_exist(client, table):
     assert response.json['title'] == 'Bad Request'
     assert response.json['detail'] == 'Some requested scenes could not be found: ' \
                                       'S1A_IW_SLC__1SDV_20200610T173646_20200610T173704_032958_03D14C_5F2A'
+
+
+def test_submit_job_hyp3_unavailable(client):
+    batch = [
+        make_job()
+    ]
```

Comment:
I think `submit_batch` creates this by default, can just call `submit_batch(client)`